ProtocolHandler.pm

```diff
@@ -268,11 +269,26 @@ sub getMetadataFor {
 	}
 
 	$meta->{title} = Plugins::Qobuz::API::Common->addVersionToTitle($meta);
-	
-	if ($meta->{work}) {
-		$meta->{title} = $meta->{work} . ': ' . $meta->{title};
-	}
 
+	# user pref is for enhanced classical music display, and we have a classical release	
+	if ( $meta->{isClassique} ) {
+		# if the title doesn't already contain the work text
+		if ( $meta->{work} && index($meta->{title},$meta->{work}) == -1 ) {
+			$meta->{title} =~ s/${\(split " ", $meta->{composer})[-1]}:\s*// if $meta->{composer};
+			my $simpleWork = lc(unidecode($meta->{work}));
+			$simpleWork =~ s/\W//g;
+			my $simpleTitle = lc(unidecode($meta->{title}));
```

Comment:
You could use `Slim::Utils::Text::ignoreCaseArticles('...', 1)`. This would also take into account the user's "ignore articles" setting.

Reply:
Done.